docs/input/docs/getting-started/whystrategypattern.md

```diff
@@ -0,0 +1,22 @@
+---
+Order: 10
+Title: Why does BBT.StrategyPattern exist?
+Description: Introduction to BBT.StrategyPattern, what problems are solved and how it is distinctive from other implementations of strategy pattern.
+---
+
+## Introduce structure to your code by using BBT.StrategyPattern
```

Comment:
Also document the fact that using one library based strategy pattern implementation let's you reuse the strategies across multiple projects and libraries if neccessary.

Reply:
This would be true for every infrastructure?